.github/workflows/docs.yml

```diff
@@ -0,0 +1,45 @@
+name: Documentation
+on:
+  push:
+    branches: [release/*, develop]
+  pull_request:
+    branches: [release/*, develop]
```

Comment:
We don't want to update documentation on new PRs.

Reply:
Yep, true